e2e-tests/tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+"""Pytest configuration for e2e tests."""
+import os
+import urllib.error
+import urllib.request
+
+import pytest
+
+
+def pytest_addoption(parser):
+    """Add custom command line options."""
+    parser.addoption(
+        "--run-ollama",
+        action="store_true",
+        default=False,
+        help="Run Ollama integration tests (requires Ollama server running)"
+    )
+
+
+def pytest_configure(config):
+    """Configure pytest with custom markers and settings."""
+    config.addinivalue_line(
+        "markers",
+        "ollama: mark test as requiring Ollama server"
+    )
```

Comment:
1. Why do we need too add the marker and cli option? Can't it be done with annotating with @pytest.mark + running with `pytest -m` 
2. Why is it optional, if e2e should start the whole cluster anyway? Either way, locally if you do not want it to run it can fail anyway